test/functional/interface_usdt_net.py

```diff
@@ -77,12 +112,130 @@
     outbound_messages.perf_submit(ctx, &msg, sizeof(msg));
     return 0;
 };
+
+BPF_PERF_OUTPUT(inbound_connections);
+int trace_inbound_connection(struct pt_regs *ctx) {
+    struct NewConnection inbound = {};
+    uint64_t conn_type_pointer;
+    uint64_t address_pointer;
+    bpf_usdt_readarg(1, ctx, &inbound.conn.id);
+    bpf_usdt_readarg(2, ctx, &address_pointer);
+    bpf_usdt_readarg(3, ctx, &conn_type_pointer);
+    bpf_usdt_readarg(4, ctx, &inbound.conn.network);
+    bpf_usdt_readarg(5, ctx, &inbound.existing);
+    bpf_probe_read_user_str(&inbound.conn.addr, sizeof(inbound.conn.addr), (void *) address_pointer);
+    bpf_probe_read_user_str(&inbound.conn.type, sizeof(inbound.conn.type), (void *) conn_type_pointer);
```

Comment:
Non blocker comment: this cast from `uint64_t` to `(void*)` looks odd, maybe it will produce compiler warning in some cases? Also, on 32 bit systems that should be `uint32_t`. Can these variables be declared as `void*` instead? Like:

```suggestion
    void* conn_type_pointer;
    void* address_pointer;
    bpf_usdt_readarg(1, ctx, &inbound.conn.id);
    bpf_usdt_readarg(2, ctx, &address_pointer);
    bpf_usdt_readarg(3, ctx, &conn_type_pointer);
    bpf_usdt_readarg(4, ctx, &inbound.conn.network);
    bpf_usdt_readarg(5, ctx, &inbound.existing);
    bpf_probe_read_user_str(&inbound.conn.addr, sizeof(inbound.conn.addr), address_pointer);
    bpf_probe_read_user_str(&inbound.conn.type, sizeof(inbound.conn.type), conn_type_pointer);
```

Reply:
Thanks! Changed the declaration from `uint64_t` to `void*`.